### ISSUE TITLE
Allow creating a gate by dragging

### ISSUE THREAD

**Issue by rsehgal** [rsehgal/barcQ#2]
Currently user can  create a gate by clicking on the available gates.
But it will be better if we allow to create a gate by dragging them.
So we have to add logic to add a gate on drag event instead of click event.

------------------------------------------------------------------------------------------------
NOTE: Steps to start the work.
1) Clone the repository
2) Create a new feature branch, and work in your branch, once you think that the feature is complete then create a pull request to merge your work to master.

Once i received the merge request, i will review the code and if found satisfactory then it  will merged into the master
------------------------------------------------------------------------------------------------

**Comment by rsehgal** [comment]
The issue is already handled by Bharti in her issue https://github.com/rsehgal/barcQ/issues/34 .
Hence closing the issue
